### PR TITLE
SW-4761 Move species view into scenes directory

### PR DESCRIPTION
**Pull request by nickgraz**
SW-4761 Move species view into scenes directory

Load species in species view if they haven't already been loaded (eventually this will be removed from the top level app component

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2065?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #2050** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2050?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2061** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2061?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #2062** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2062?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #2063** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2063?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #2064** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2064?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #2065** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2065?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
              * **PR #2066** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2066?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #2067** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2067?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2065?utm_source=stack-comment).

File: src/scenes/Species/AddSpeciesModal.tsx [hunk 1]
```diff
@@ -16,11 +16,11 @@ import {
...
-import Button from '../common/button/Button';
-import Checkbox from '../common/Checkbox';
-import DialogBox from '../common/DialogBox/DialogBox';
-import Select from '../common/Select/Select';
-import TextField from '../common/Textfield/Textfield';
+import Button from '../../components/common/button/Button';
+import Checkbox from '../../components/common/Checkbox';
+import DialogBox from '../../components/common/DialogBox/DialogBox';
+import Select from '../../components/common/Select/Select';
+import TextField from '../../components/common/Textfield/Textfield';
```
Comment (karthikbtf) [inline src/scenes/Species/AddSpeciesModal.tsx:23]: `src/components`

**Review by karthikbtf — APPROVED** [review]
some comments, clean up relative paths, etc. at your discretion

**Comment by sgrimm** [comment]
### Merge activity

* **Jan 25, 3:38 PM**: @sgrimm started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2065).

**Pull request merged**